server/data/licenceApiClient.ts

```diff
@@ -446,4 +446,13 @@ export default class LicenceApiClient extends RestClient {
       data: offenderDetails,
     })
   }
+
+  async getParentLicenceOrSelf(licenceId: string, user: User): Promise<Licence> {
```

Comment:
Is there a reason this was moved here from LicenceService?

Reply:
Yes, In order to remove circular dependency move getParentOrSelfAdditionalLicenceConditions this functionality to conditionalService we decided to move(Andy suggested)